exe-common/src/sync.rs

```diff
@@ -354,7 +354,7 @@ fn finish_epoch(
 ) -> Result<()> {
     let epoch_id = epoch_writer_state.epoch_id;
     let (packhash, index) = pack::packwriter_finalize(&storage.config, epoch_writer_state.writer);
-    let (lookup, tmpfile) = pack::create_index(&storage, &index);
+    let (lookup, tmpfile) = pack::create_index(&storage, &index, epoch_id as u32);
```

Comment:
Gotta pass `epoch_id` to index creation here, to store it as `ordinal` parameter, to be able to sort lookups later.